metadata.json

```diff
@@ -83,7 +83,7 @@
   "requirements": [
     {
       "name": "puppet",
-      "version_requirement": ">= 6.0.0"
+      "version_requirement": ">= 6.1.0 < 8.0.0"
```

Comment:
Should be handled by:
https://github.com/voxpupuli/puppet_metadata/commit/491aa993479260ef37b9a926ab4b3aa07c1e3fba

The requirement for puppet_metadata is too low.

Reply:
Ah that's why itnsticked in endless loop.
Isn't it better to constraint it?

Also I expected the modulesync to bring the Gemfile up-to-date. Probably I missed something?

Reply:
>  Isn't it better to constraint it?

Yes :-)  It was just FYI :-)